### PR TITLE
Vedant profile added

### PR DESCRIPTION
**Pull request by vedantpople4**

### CODE REVIEW

File: Readmes/Vedant-Pople/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        </div>
+      
+<!--START_SECTION:activity-->
+1. 🗣 Commented on [#5](https://github.com/AccessibleForAll/Resources/issues/5) in [AccessibleForAll/Resources](https://github.com/AccessibleForAll/Resources)
+2. ❗️ Opened issue [#10](https://github.com/AccessibleForAll/Support/issues/10) in [AccessibleForAll/Support](https://github.com/AccessibleForAll/Support)
+3. ❗️ Opened issue [#422](https://github.com/Devs-Dungeon/support/issues/422) in [Devs-Dungeon/support](https://github.com/Devs-Dungeon/support)
+4. 💪 Opened PR [#8](https://github.com/Anzo52/concalcjava/pull/8) in [Anzo52/concalcjava](https://github.com/Anzo52/concalcjava)
+5. 🗣 Commented on [#1](https://github.com/meftunca/TensorFlow-Course/issues/1) in [meftunca/TensorFlow-Course](https://github.com/meftunca/TensorFlow-Course)
+<!--END_SECTION:activity-->
+
```
Comment (Panquesito7) [inline Readmes/Vedant-Pople/README.md:114]: There's an extra line here.
```suggestion
```

File: Readmes/Vedant-Pople/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+<a href = "https://twitter.com/vedantpople"><img src="https://img.icons8.com/fluent/48/000000/twitter.png"/></a>
+<a href = "https://www.instagram.com/vedant.pople"><img src="https://img.icons8.com/fluent/48/000000/instagram-new.png"/></a>
+<!-- <a  href ="https://app.slack.com/client/T4N3CG75G/learning-slack/user_profile/U02A71HGFC2"><img src ="https://img.icons8.com/color/48/000000/slack-new.png"></a> -->
+<!-- <a href = "" ><img src = ""> -->
+<!-- <a href =""><img src="https://img.icons8.com/cute-clipart/48/000000/discord-logo.png"/> -->
+
+
+</p>
+
+## ❤ Views and Followers
```
Comment (Panquesito7) [inline Readmes/Vedant-Pople/README.md:101]:
```suggestion
## ❤ Views and Followers

```

File: Readmes/Vedant-Pople/README.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+ VP of Educational Operations at **C4-Projects**.
+    
+ Administrative Head at **ISA-VIT** (2020-2021)
+ 
+ Data Analysis Intern at **MedTourEasy** (2021)
+
+ Winner, Sponsor Track at **Code2Create 5.0**
+
+
+## 🚀 Languages and Tools:
```
Comment (Panquesito7) [inline Readmes/Vedant-Pople/README.md:43]: I recommend removing the `:` here. What do you think?

```suggestion
## 🚀 Languages and Tools
```

File: Readmes/Vedant-Pople/README.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    <a href="https://github.com/vedantpople4/github-readme-stats"><img alt="Vedant Pople's Github Stats" src="https://github-readme-stats.vercel.app/api?username=vedantpople4&show_icons=true&count_private=true&theme=react&hide_border=true&bg_color=0D1117" /></a>
+             
+  <a href="https://github.com/vedantpople4/github-readme-stats"><img alt="Vedant's Top Languages" src="https://github-readme-stats.vercel.app/api/top-langs/?username=vedantpople4&langs_count=8&count_private=true&layout=compact&theme=react&hide_border=true&bg_color=0D1117" /></a>
+  <br/>
+  <b>Note:</b> Top languages is only a metric of the languages my public code consists of and doesn't reflect experience or skill level.
+
+<br/>
+<br/>
+
+## Connect with me:
```
Comment (Panquesito7) [inline Readmes/Vedant-Pople/README.md:88]: I recommend removing the `:` here. What do you think?
```suggestion
## Connect with me

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Good readme! 🚀

File: Readmes/Vedant-Pople/README.md (hunk 1 above)
Comment (vedantpople4) [reply Readmes/Vedant-Pople/README.md:114]: Done

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Greeshma2903 — APPROVED** [review]

**Comment by Greeshma2903** [comment]
Thank you for contributing! 💯

**Pull request merged**